### PR TITLE
Fix nightly build

### PR DESCRIPTION
**Pull request by romac**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #![deny(
-    missing_docs,
```
Comment (FredericJacobs) [inline src/lib.rs:2]: I'm slightly confused as to why reordering the parameters fixes the nightly build. Do you have an explanation or reference?

Comment (romac) [reply src/lib.rs:2]: It does not, it's just a byproduct of me trying out a few things and putting the flag back in the wrong place. Line 28 is where the fix actually is.

Comment (FredericJacobs) [reply src/lib.rs:2]: Gotcha.

**Review by FredericJacobs — APPROVED** [review]
LGTM. Since protobuf is automatically generated code, we can discard unused qualifications in it.

**Pull request merged**